plans/tier1.fmf

```diff
@@ -204,3 +204,15 @@ discover+:
   - name: reboot after conversion
     how: ansible
     playbook: tests/ansible_collections/roles/reboot/main.yml
+
+/removed_pkgs_centos_85:
+  adjust:
+    enabled: false
+    when: >
+      distro != centos-8
+  discover+:
+    test: removed-pkgs-centos-85
```

Comment:
In tier1 we use `checks-after-conversion` as a final test. Please add your `removed-pkgs-centos-85` to a preparation phase.

Reply:
I would rather leave it in separate folder because of the different repos, which are added in the ansible part.

Reply:
@kokesak Anyway, I've changed the final test, now it should be fine.